src/test/java/teammates/test/cases/logic/StudentsLogicTest.java

```diff
@@ -463,9 +463,11 @@ private void testEnrollLinesChecking() throws Exception {
         String lineWithInvalidTeamNameAndStudentNameAndEmail =
                 invalidTeamName + "|" + invalidStudentName + "|" + "invalid_email.tmt";
 
-        enrollLines = headerLine + Const.EOL + lineWithInvalidTeamName + Const.EOL + lineWithInvalidStudentName
-                    + Const.EOL + lineWithInvalidEmail + Const.EOL + lineWithInvalidStudentNameAndEmail + Const.EOL
-                    + lineWithInvalidTeamNameAndEmail + Const.EOL + lineWithInvalidTeamNameAndStudentNameAndEmail;
+        enrollLines = headerLine + System.lineSeparator() + lineWithInvalidTeamName + System.lineSeparator()
```

Comment:
Not your fault, but this is a little hard. Let's break after every `System.lineSeparator()`.